oeps/oep-0019-bp-developer-documentation.rst

```diff
@@ -52,6 +52,21 @@ This proposal focuses on long-term developer *Documentation* needs. It does not
 Decisions
 *********
 
+* We will use `reStructuredText (rST)`_ for all developer documentation (including in docstrings).  See the :ref:`Format Rationale` section below for more info.
+
+* The `Developer Docs`_ section of docs.edx.org will serve as the index for all repo specific and system wide docs.
+
+* System wide decisions, best practices and processes will live in the `open-edx-proposals`_ repository.
+
+* ADRs, How Tos and other repo global docs will live in a ``/docs`` folder in a repo.
```

Comment:
1. "repo global docs" => "repo specific docs"? Note: the term "global" makes me think about global-global, not local-global. So, can we avoid it somehow?
2. Note: When we've done some cross-repo features that have a potential single repo that could serve as the root, we have also preferred to choose that repo rather than using edx-developer-docs. For example, we chose `edx-toggles` for toggles documentation indexing, which includes work in code-annotations, and edx-lint. I bring this up, because these docs aren't 100% "repo specific", but not sure what term to give this.

Reply:
1. Will update the wording.
2. I'm not sure what to do about this yet but probably something we don't need to address in this PR.